africanlii/templates/peachjam/member_state_detail.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'peachjam/layouts/document_list.html' %}
+{% load i18n %}
+{% block title %}{{ member_state }}{% endblock %}
+{% block breadcrumbs %}
+  <div class="container">
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <!-- djlint:off -->
+          <a href="/au/#member-state"><!-- djlint:on -->{% trans 'Member States' %}</a>
+        </li>
+        <li class="breadcrumb-item">
+          <a href="{% url 'member_state_detail_view' member_state.pk %}">{{ member_state }}</a>
+        </li>
+      </ol>
+    </nav>
+  </div>
+{% endblock %}
+{% block entity-profile %}
+  {% with entity_profile=member_state.entity_profile.first entity_profile_title=member_state.name %}
+    {% if entity_profile %}
+      <div class="mt-3">{% include 'peachjam/_entity_profile.html' %}</div>
+    {% endif %}
+  {% endwith %}
+{% endblock %}
+{% block page-header %}
+  <h1 class="mt-4">{{ member_state }}</h1>
+  {% include 'peachjam/_document_count.html' %}
+  {% include 'peachjam/_years_list.html' with showallyears=True %}
+{% endblock %}
+{% block content %}
+  {% if documents %}
+    {% include 'peachjam/_judgment_table.html' %}
```

Comment:
I need to work out what we show here for ratified instruments.

Reply:
For now, let's keep the list of ratified instruments in the Member State detail page simple: a table listing the document title (a link to the document) and the ratification date, ordered alphabetically. No facets or pagination necessary.